client/components/product-expiration/index.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+interface Moment {
+	isValid: () => boolean;
+	format: ( arg0: string ) => string;
+	diff: ( arg0: Date ) => number;
+}
+
+export interface Props {
+	expiryDateMoment: Moment;
+	purchaseDateMoment: Moment;
+	isRefundable: boolean;
+	dateFormat: string;
+	translate: ( arg0: string, arg1: object ) => string;
+}
+
+class ProductExpiration extends React.PureComponent< Props > {
+	static defaultProps = {
+		expiryDateMoment: null,
+		purchaseDateMoment: null,
+		isRefundable: false,
+		dateFormat: 'LL',
+	};
+
+	render() {
```

Comment:
Seems like a wonderful component to have tests for. Care to add some tests?

Reply:
Done in 006b6b03f7937676d3db3dcbca652f8a7cd73b45!